### PR TITLE
changed ingress values to more configurable syntax

### PR DESCRIPTION
**Pull request by HaveFun83**
This PR make the ingress value more configurable as it use the default ingress syntax.
Tested within our own cluster.

### CODE REVIEW

File: charts/codimd/README.md [hunk 1]
```diff
@@ -68,10 +68,10 @@ If you want use ingress, please set `service.type` to be `ClusterIP`
 | service.port                  | Service HTTP port                     | `80`           |
 | service.externalTrafficPolicy | Service externalTrafficPolicy         | `nil`           |
 | service.loadBalancerIP        | Service loadBalancerIP                | `nil`           |
-| ingress.enabled               | Enable ingress controller resource    | `false`        |
-| ingress.annotation            | Ingress annotations                   | `nil`          |
-| ingress.hostname              | Default host for the ingress resource | `nil`          |
-| ingress.tlsSecret             | TLS Secret Name                       | `nil`          |
+| ingress.enabled               | If true, Ingress will be created      | `false`         |
+| ingress.annotations`          | Ingress annotations                   | `nil`           |
```
Comment (Yukaii) [inline charts/codimd/README.md:72]: Is that backtick a typo?

Comment (HaveFun83) [reply charts/codimd/README.md]: fixed

**Comment by HaveFun83** [comment]
@a60814billy any news here?

**Comment by HaveFun83** [comment]
> thanks @HaveFun83, LGTM.

Wow this was fast :D thanks

**Comment by HaveFun83** [comment]
@a60814billy can you also release the new chart version on https://helm.codimd.dev/ ?

**Comment by a60814billy** [comment]
Hi @HaveFun83, the new helm chart has already deployed on https://helm.codimd.dev/.

**Pull request merged**